docs/guides/blocks/cards.md

```diff
@@ -0,0 +1,224 @@
+---
+nav: Cards
+description: Display structured information as rich cards.
+---
+
+# Preview content with rich cards
+
+Pack formulas can return structured data as [objects][data_types_objects], allowing a single formula to return multiple values at once. By default these objects are presented as "mentions", shown as chips in the document that you can hover over to get the full set of information.
+
+<img src="../../../images/cards_demo_mention.png" srcset="../../../images/cards_demo_mention_2x.png 2x" alt="Schema shown as a mention">
+
+Cards are an alternative, more visual display format for objects that allow the user to easily consume key information. Additionally URLs pasted into a document can be automatically shown as cards, providing an easy way for users to discover and use your Pack.
```

Comment:
maybe link to a matchers section for more info about how they get automatically replaced?

Reply:
This is meant to be a soft intro into the topic, and I think I'd rather have the user read on vs jump ahead to that advanced topic. It's on the same page, so I think there is a high chance they'll discover it.

Reply:
okay that's fair